apps/09todo-jquery/script.js

```diff
@@ -1,5 +1,39 @@
 'use strict';
 
 $(document).ready(function() {
-  // App logic goes here
+    $('form').submit(function(event) { // target the form, and put a submit event listener on it. We can capture and pass in the event for us to use
+
+        event.preventDefault(); // this tells the browser to "prevent" the "default" "event" from occuring (sending and refreshing)
+        console.log($(this)); // let's log out our "this" context (our form element) as a jQuery object
+        var todoText = $(this).find('input[type="text"]').val();
+        //.val if empty will be "" or could have taken the falsey approach --> !todoText
+        if (todoText === "") {
+            return;
+        }
+
+        $('#todo-list').append('<li> <input type ="checkbox" name="test" value="test"/>' + todoText + '<button class ="done"></button></li>');
+
+        $('body').on('click', 'button', function(event) {
+            $(this).parent().remove();
+
+
+        });
+
+        //this is the button in the scenario above.
+
+        $(this).find('input[type="text"]').val('');
+
+        $('ul').sortable(); //could also put draggable here. Interacting with the DOM
```

Comment:
I would put this logic at the top of your code so it's not re-rendered after every submit.